### PR TITLE
[DX-1437] Replace authorise with authorize in godoc comments

### PR DESCRIPTION
**Pull request by dcs3spp**
## Description
Replace _authorise_ with _authorize_ for [PR](https://github.com/TykTechnologies/tyk-docs/pull/4937) raised in docs by @Roeegg2 

## Motivation and Context

uk -> us spelling update

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Docs update

## Checklist

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). If pulling from your own
      fork, don't request your `master`!
- [ ] Make sure you are making a pull request against the **`master` branch** (left side). Also, you should start
      *your branch* off *our latest `master`*.
- [x] My change requires a change to the documentation.
  - [ ] If you've changed APIs, describe what needs to be updated in the documentation.
- [x] I have updated the documentation accordingly.
- [ ] Modules and vendor dependencies have been updated; run `go mod tidy && go mod vendor`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] Check your code additions will not fail linting checks:
  - [ ] `go fmt -s`
  - [ ] `go vet`

### CODE REVIEW

File: pumps/hybrid.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ type HybridPumpConf struct {
 	ConnectionString string `mapstructure:"connection_string"`
 	// Your organisation ID to connect to the MDCB installation.
```
Comment (sredxny) [inline pumps/hybrid.go:71]: @dcs3spp should organisation be changed to organization?

**Review by sredxny — CHANGES_REQUESTED** [review]

**Review by letzya — DISMISSED** [review]

File: pumps/hybrid.go (hunk 1 above)
Comment (letzya) [reply pumps/hybrid.go:71]: correct. We haven't done this word yet but we can add it @dcs3spp

Comment (dcs3spp) [reply pumps/hybrid.go]: @letzya @sredxny I've updated organisation to organization

**Review by letzya — APPROVED** [review]

**Review by letzya — APPROVED** [review]

**Comment by dcs3spp** [comment]
@sredxny Why do some of the Release CI checks fail for api-tests fail with error _dependency failed to start: container auto-gateway-checker-1 is unhealthy_

**Review by sredxny — APPROVED** [review]

**Comment by dcs3spp** [comment]
@sredxny Tests still failing after rerun. I do not have permission to force merge into master branch

**Comment by dcs3spp** [comment]
@buger @sredxny and I are unable to merge due to _Release / api-tests_ CI checks failing. We are both unable to override force merging. How can we merge or can you force merge and then `/release to release-10`?

**Pull request merged**